bootstrap/012_task_log.sql

```diff
@@ -0,0 +1,53 @@
+
+create or replace function internal.task_run_id()
+returns text
+AS
+$$
+     SYSTEM$TASK_RUNTIME_INFO('CURRENT_TASK_GRAPH_RUN_GROUP_ID')
+$$;
+
+-- Generic table that tasks should record their execution into.
+CREATE TABLE INTERNAL.TASK_LOG IF NOT EXISTS (task_start timestamp_ltz, success boolean, task_name varchar, object_name varchar,
+    input variant, output variant, task_finish timestamp_ltz, task_run_id text, query_id text);
```

Comment:
I hadn't included the min/max range of data materialized (e.g. we have query history data from 2023-01-01 00:00:00 to 2023-04-30 05:30:00). because I thought this would be easy to dynamically compute after the fact.

However, it was not as simple as I expected to dynamically construct and run the `select min(..), max(...) from ..` queries. I'm leaning towards to including two new columns (range_min, range_max) into the task_log table.

Reply:
I thoght the min and max were included in the output variant?

Reply:
updated `output` so it includes `range_min` and `range_max`